draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1163,111 +1090,66 @@ initiating and the responding agent.
+address, this value SHOULD be set to 65535.  If there
 are multiple candidates for a particular component for a particular
 media stream that have the same type, the local preference MUST be
-unique for each one. In this specification, this only happens for
-multihomed hosts or if an agent is using multiple TURN servers.  If a
-host is multihomed because it is dual-stack, the local preference
+unique for each one. If an agent is dual-stack, the IP address preference
 should be set according to the current best practice described in <xref
 target="I-D.ietf-ice-dualstack-fairness"/>.</t>
 
 
-<t>The
-component ID is the component ID for the candidate, and MUST be
-between 1 and 256 inclusive.
-</t>
+<t>The component ID MUST be an integer between 1 and 256 inclusive.</t>
 
 
 </section>
 
 <section anchor="sec-guidelines" title="Guidelines for Choosing Type and Local Preferences">
 
-<t>One criterion for selection of the type and local preference values
-is the use of a media intermediary, such as a TURN server, a tunnel
-service such as VPN server, or NAT.  With a media intermediary, if
-media is sent to that candidate, it will first transit the media
-intermediary before being received.  Relayed candidates are one type
-of candidate that involves a media intermediary.  Another are host
-candidates obtained from a VPN interface.  When media is transited
-through a media intermediary, it can have a positive or negative
-effect on the latency between transmission and reception.  It may or
-may not increase the packet losses, because of the additional router
-hops that may be taken.  It may increase the cost of providing
-service, since media will be routed in and right back out of a media
-intermediary run by a provider.  If these concerns are important, the
-type preference for relayed candidates must be carefully chosen. The
-RECOMMENDED values are 126 for host candidates, 100 for server
-reflexive candidates, 110 for peer reflexive candidates, and 0 for
-relayed candidates.</t>
-
-<t>Furthermore, if an agent is multihomed and has multiple IP
-addresses, the recommendation in <xref target=
-"I-D.ietf-ice-dualstack-fairness"/> should be followed.  If multiple
+<t>The RECOMMENDED values for type preferences are 126 for host
+candidates, 110 for peer reflexivec andidates, 100 for server
+reflexive candidates, and 0 for relayed candidates.</t>
+
+<t>If an agent is multihomed and has multiple IP
+addresses, the recommendations in <xref target=
+"I-D.ietf-ice-dualstack-fairness"/> SHOULD be followed.  If multiple
 TURN servers are used, local priorities for the candidates obtained
 from the TURN servers are chosen in a similar fashion as for
-multihomed local candidates: the local preference value is used to
+multihomed local candidates: the IP address preference value is used to
 indicate a preference among different servers but the preference MUST
 be unique for each one.</t>
 
-<t>Another criterion for selection of preferences is IP address
```

Comment:
I still think this is irrelevant for specifying the protocol, but I'll add it back.